activities/TankOp.activity/lib/tutorial.js

```diff
@@ -0,0 +1,114 @@
+define(["webL10n"], function (l10n) {
+	var tutorial = {};
+
+	tutorial.start = function(play) {
+		var homeSteps = [
```

Comment:
Still some indentation mix, check your editor:

![image](https://user-images.githubusercontent.com/981841/75098834-3f737d00-55bb-11ea-981f-3aade69ca242.png)

Reply:
Indention mix issue in tutorials.js is resolved.